nuxeo-distribution/nuxeo-nxr-server/src/main/resources/templates/nuxeo.defaults

```diff
@@ -48,10 +48,10 @@ opensocial.gadgets.path=/site/gadgets
 opensocial.trusted.hosts=
 
 # Clustering
-nuxeo.cluster.enabled=false
+nuxeo.cluster.enabled=
```

Comment:
maybe add a comment like: use empty value instead of false for backward compat with repository.clustering.enabled